core/src/com/unciv/ui/tilegroups/TileGroupMap.kt

```diff
@@ -162,9 +162,14 @@ class TileGroupMap<T: TileGroup>(
 
         if (worldWrap) {
 
+            // Prevent flickering when zoomed out
+            val mapWidth: Float
+            if(mapHolder.width > width) mapWidth = width - groupSize * 1.5f
```

Comment:
How could it not be? What, if the entire map is only 1 tile wide? 😕

Reply:
width - TileGroupMap.width or map width, always the same after map generation
mapHolder.width - the width of the visible area in the mapholder, decreases when zooming in, increases when zooming out

Reply:
maybe the val name should be changed to something like visibleMapWidth?

Reply:
That would be clearer, yes

Reply:
Because I can totally get how the *visible width* can be bigger than the *total width*
Also maybe add a comment like 'zoomed out so you can see entire map' for clarity

Reply:
Updated